src/main/java/teammates/ui/template/StudentProfile.java

```diff
@@ -54,4 +58,11 @@ public String getNationality() {
     public String getMoreInfo() {
         return moreInfo;
     }
+
+    // Ensures gender type is displayed in lower case
```

Comment:
This comment does not value add; why would one need to ensure that it is displayed in lower case?